datumaro/cli/contexts/project/__init__.py

```diff
@@ -399,6 +402,55 @@ def filter_command(args):
 
     return 0
 
+def build_generate_parser(parser_ctor=argparse.ArgumentParser):
+    parser = parser_ctor(help="Generate synthetic dataset",
+        description="""
+        This command is useful for making a synthetic dataset
+        with specified height, width and storing them in provided directory.|n
+        |n
+        Examples:|n
+        - Generate 300 3-channel synthetic images with H=224, W=256 and store to data_dir:|n
+        |s|s%(prog)s -o data_dir -k 300 --height 224 --width 256|n
+        """,
+        formatter_class=MultilineFormatter)
+
+    parser.add_argument('-o', '--output-dir', required=True,
+        help="Output directory to store generated dataset")
+    parser.add_argument('-k', '--count', type=int, required=True,
```

Comment:
`-k` looks strange to me. My suggestion is to keep only `--count` if `-c` is busy.

Reply:
('-k', '--count')  is aligned with datumaro style

Reply:
interesting style)